gulp/tasks/scripts.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const config = require( '../config' );
+const gulp = require( 'gulp' );
+const gulpRename = require( 'gulp-rename' );
+const webpack = require( 'webpack' );
+const webpackStream = require( 'webpack-stream' );
+
+// TODO:
+// Consolidate build scripts to call one helper function like in cfgov-refresh.
+
+/**
+ * Generate main bundle.
+ * @returns {PassThrough} A source stream.
+ */
+function exampleScripts() {
+  return gulp.src( './examples/index.js' )
+    .pipe( webpackStream( {
+      module: {
+        rules: [ {
+          use: [ {
+            loader: 'babel-loader?cacheDirectory=true',
```

Comment:
Yeah, TODO on line 10 is a poorly worded attempt at saying make this whole function shared by the two tasks, in which case this would only be referenced one time.